### PR TITLE
SRS BODBAW - modify folder structure on storage

### PR DESCRIPTION
**Pull request by lbesnard**

### CODE REVIEW

**Comment by lbesnard** [comment]
no rush to merge

File: aodndata/srs/srs_oc_bodbaw.py [hunk 1]
```diff
@@ -33,8 +33,8 @@ def dest_path(self, filepath):
         if 'absorption' in product_type:
             product_type = 'absorption'
 
-        cruise_id = m.group(2)
+        cruise_id = m.group(2).split('-')[0]  # split to get rid of the station_code within the cruise_id if exists
         year = datetime.strptime(m.group(1), '%Y%m%dT%H%M%SZ').strftime("%Y")
 
-        return os.path.join(bodbaw_dir, '{year}_cruise-{cruise}'.format(year=year, cruise=cruise_id),
+        return os.path.join(bodbaw_dir, '{cruise}'.format(year=year, cruise=cruise_id),
```
Comment (bpasquer) [inline aodndata/srs/srs_oc_bodbaw.py:39]: you can get rid of the reference to `year` for the formatting

File: test_aodndata/srs/test_srs_oc_bodbaw.py [hunk 2]
```diff
@@ -31,19 +33,39 @@ def test_netcdf(self):
...
+                                   include_regexes=[r'IMOS_SRS-OC-BODBAW_X_.*\.nc'],
+                                   check_params={'checks': ['cf'],
+                                                 'criteria': 'lenient'}
+                                   )
+
+        f = handler.file_collection[0]
+        #self.assertEqual(f.check_type, PipelineFileCheckType.NC_COMPLIANCE_CHECK)
+        self.assertEqual(f.publish_type, PipelineFilePublishType.HARVEST_UPLOAD)
+        self.assertEqual(f.dest_path,
+                         os.path.join('IMOS/SRS/OC/BODBAW/SWAIN/absorption/'
```
Comment (bpasquer) [inline test_aodndata/srs/test_srs_oc_bodbaw.py:56]: remove the extra line

**Comment by lbesnard** [comment]
just did the change

**Pull request merged**